modules/ipi-install-bmc-addressing.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-install-bmc-addressing_{context}"]
+
+= BMC addressing
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+.IPMI
+
+IPMI hosts use `ipmi://<out-of-band-ip>:<port>` and defaults to port `623` if not specified. The following example demonstrates an IPMI configuration within the `install-config.yaml` file.
+
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: ipmi://<out-of-band-ip>
+          username: <user>
+          password: <password>
+----
+
+.RedFish
+
+To enable RedFish, use `redfish://` (or `redfish+http://` to disable TLS). The installer requires both the hostname (or IP address) and the path to the system ID. The following example demonstrates a RedFish configuration within the `install-config.yaml` file.
+
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish://<out-of-band-ip>/redfish/v1/Systems/1
+          username: <user>
+          password: <password>
+----
+
+While it is recommended to have a certificate of authority for the
+out-of-band management addresses, you must include `disableCertificateVerification: True` in the `bmc` configuration if using self-signed certificates. The following example demonstrates a RedFish configuration using the `disableCertificateVerification: True` configuration parameter within the `install-config.yaml` file.
+
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish://<out-of-band-ip>/redfish/v1/Systems/1
+          username: <user>
+          password: <password>
+          disableCertificateVerification: True
+----
+
+NOTE: Currently RedFish is only supported on HPE hardware for IPI on Bare metal deployments. We are working with other vendors to enable RedFish capabilities across the board.
```

Comment:
[NOTE]
    ====
    Currently, RedFish is only supported on HPE hardware for IPI on Bare metal deployments.
    ====